hack/update-api-reference-docs.sh

```diff
@@ -37,12 +47,40 @@ echo "Reading swagger spec from: ${SWAGGER_PATH}"
 mkdir -p $V1_PATH
 mkdir -p $V1BETA1_PATH
 
-docker run --rm -v $V1_PATH:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v3 \
+docker run -u $(id -u) --rm -v $V1_TEMP_PATH:/output:z -v ${SWAGGER_PATH}:/swagger-source:z gcr.io/google_containers/gen-swagger-docs:v4 \
```

Comment:
Test runs don't share workspaces (the jenkins dir where the repo gets cloned down, from which dockerized tests run, mapping in REPO_DIR), so there's no race.